Library/Homebrew/cleanup.rb

```diff
@@ -1,6 +1,71 @@
 require "utils/bottles"
 require "formula"
 
+module CleanupRefinement
```

Comment:
Is this replacing monkeypatching or just makes the code below a bit nicer?

Reply:
Just to make the actual cleanup logic easier to read.